doc/man3/OCSP_sendreq_new.pod

```diff
@@ -92,14 +90,8 @@ a Host header for B<ocsp.com> you would call:
 
  OCSP_REQ_CTX_add1_header(ctx, "Host", "ocsp.com");
 
-If OCSP_sendreq_nbio() indicates an operation should be retried the
-corresponding BIO can be examined to determine which operation (read or
-write) should be retried and appropriate action taken (for example a select()
-call on the underlying socket).
-
-OCSP_sendreq_bio() does not support retries and so cannot handle non-blocking
-I/O efficiently. It is retained for compatibility and its use in new
-applications is not recommended.
+OCSP_sendreq_bio() does not support timeout nor setting extra headers.
+It is retained for compatibility.
```

Comment:
If you are intending to encourage some other function instead you should state what the preferred function is.

Reply:
I just extended that existing sentence so that statement should have been given already earlier.
T rectify this I've now added
```
Better use B<OCSP_sendreq_nbio()> instead.
```